tools/translationsreport/app/src/App.js

```diff
@@ -0,0 +1,81 @@
+import { useState, useEffect } from 'react'
+
+import { Grid, Segment, Header, Container, Divider } from 'semantic-ui-react'
+
+import StringsList from "./StringsList.js"
+
+// Data from latest main
+const DEFAULT_DATA_URL = "TODO"
```

Comment:
@strseb Not sure what to put here, how do I know the index URL for main that we created? I know from the taskgraph report that it will contain `index.mozillavpn.v2.mozilla-vpn-client.branch.main.latest.tools.translation-report`... but is the full thing?

Reply:
No index but the rest is full. i.e `https://firefox-ci-tc.services.mozilla.com/tasks/index/mozillavpn.v2.mozilla-vpn-client.branch.main.latest.addons/stage`

Reply:
```suggestion
const DEFAULT_DATA_URL = "https://firefox-ci-tc.services.mozilla.com/tasks/index/mozillavpn.v2.mozilla-vpn-client.branch.main.latest.tools.translation-report"
```